File: Grafos/Arista.java
public class Arista{
	public int costo;
	public Vertice v,u;

	public Arista(Vertice v, Vertice u){
		this.v = v;
		this.u = u;
	}

	public Arista(Vertice v, Vertice u, int costo){
		this.v = v;
		this.u = u;
		this.costo = costo;
	}

	public void imprimirArista(){
		System.out.println("Arista: " + this.v.nombre + " <-- " + this.costo + " --> " + this.u.nombre);
	}
}

File: Python x EDAII/Grafos/Arista.py
from Vertice import Vertice

class Arista:
	costo = 0
	v = Vertice(" ")
	u = Vertice(" ")

	def __init__(self,v,u,costo):
		'Constructor de la clase'
		self.costo = costo
		self.v = v
		self.u = u

	def imprimirArista(self):
		'Imprime las características de la arista'
		print("Arista:", self.v.nombre," <-- ",self.costo," --> ",self.u.nombre)